tools/cpp/windows_cc_configure.bzl

```diff
@@ -124,13 +124,13 @@ def find_vc_path(repository_ctx):
         _auto_configure_warning_maybe(repository_ctx, "Visual C++ build tools found at %s" % vc_dir)
         return vc_dir
 
-    # 3. User might clean up all environment variables, if so looking for Visual C++ through registry.
-    # Works for all VS versions, including Visual Studio 2017.
+    # 3. User might have purged all environment variables. If so, look for Visual C++ in registry.
+    # Works for Visual Studio 2017 and older. (Does not work for Visual Studio 2019.)
```

Comment:
I guess this didn't work because VS 2019 is still in preview? We should check later when they have the official release. Can you put a comment here?

Reply:
Done.